ecs-cli/modules/clients/aws/cloudformation/cluster_template.go

```diff
@@ -304,6 +304,8 @@ var cluster_template = `
       "Condition": "CreateVpcResources",
       "Type": "AWS::EC2::VPC",
       "Properties": {
+        "EnableDnsSupport" : true,
+        "EnableDnsHostnames" : true,
```

Comment:
Because `--enable-service-discovery`  flag is only on `compose service up`, not cluster up. We could add it on cluster up as well, but I think that's not as nice of a UX. Then a customer has to know that they want to use Service Discovery when they create their cluster. 

Instead, the idea is that all clusters will work with Service Discovery. This mirrors the ECS Console, which creates clusters with both of these set to true.

Reply:
Oh, right. Good point!